docs/reference-docs/do-while-task.md

```diff
@@ -3,82 +3,86 @@ sidebar_position: 1
 ---
 
 # Do While
+
 ```json
 "type" : "DO_WHILE"
 ```
+
 ## Introduction
-Sequentially execute a list of task as long as a condition is true. 
+
+Sequentially execute a list of task as long as a condition is true.
 The list of tasks is executed first, before the condition is checked (even for the first iteration).
 
-When scheduled, each task of this loop will see its `taskReferenceName` concatenated with __i, with i being the iteration number, starting at 1. Warning: taskReferenceName containing arithmetic operators must not be used.
+When scheduled, each task of this loop will see its `taskReferenceName` concatenated with \_\_i, with i being the iteration number, starting at 1. Warning: taskReferenceName containing arithmetic operators must not be used.
```

Comment:
We don't need to escape the characters in the docs - it adds confusion to the reader (IMO)